cloudhub-2/modules/ROOT/pages/ps-gather-setup-info.adoc

```diff
@@ -373,7 +373,10 @@ The CIDR block must be unique across all private network connections.
 
 MuleSoft recommends using subnets from standard RFC 1918 addresses, such as `10.0.0.0/8`, `172.16.0.0/12`, and `192.168.0.0/16`.
 
-You cannot use the following reserved CIDR blocks:
+[[reserved-cidr]]
+==== Reserved and Unusable CIDR
```

Comment:
Unusable CIDR.

We don't want to mix reserved and unusable CIDRs.

Reply:
```suggestion
==== Unusable CIDR
```